conf/weblate.service

```diff
@@ -22,7 +22,7 @@ ExecStart=/usr/bin/uwsgi \
 Restart=on-failure
 KillSignal=SIGQUIT
 Type=notify
-StandardError=syslog
+StandardError=inherit
```

Comment:
Hmmmm not sure what’s the signification of that ?

Reply:
When service fails, it keeps saying that syslog is deprecated (or I misunderstood?)